formidable/views.py

```diff
@@ -242,7 +242,7 @@ def get(self, request, **kwargs):
         try:
             formidable = self.get_formidable_object(kwargs)
         except Formidable.DoesNotExist:
-            raise exceptions.Http404()
+            raise exceptions.NotFound()
```

Comment:
Nitpick: I think we can only return the reference of the exception here `raise exceptions.NotFound`, not an instance.

Reply:
I don't mind , just want to know why a thing is better than another :)

Reply:
Dunno. I've always told that if you're not passing any argument to the exception's constructor, then there is no point of raising an instance, but a reference to the exception class suffice. It was a nitpick anyway. We can leave it as is 😉

Reply:
Finally, I think the code is actually transforming the class to an instance automatically.
In the final exception handler, we can see something like this:

https://github.com/encode/django-rest-framework/blob/master/rest_framework/views.py#L433

But, `isinstance` does not work on class itself.

```python
```